src/views/Dashboard/components/BoxWithDisplay.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useContext, useMemo } from "react";
+import styled, { ThemeContext } from "styled-components";
+
+export interface BoxProps {
+  alignItems?: "baseline" | "center" | "flex-end" | "flex-start";
+  children?: React.ReactNode;
+  column?: boolean;
+  flex?: number | string;
+  height?: number | string;
+  justifyContent?: "center" | "flex-end" | "flex-start" | "space-around" | "space-between";
+  margin?: number;
+  marginBottom?: number;
+  marginHorizontal?: number;
+  marginLeft?: number;
+  marginRight?: number;
+  marginTop?: number;
+  marginVertical?: number;
+  maxHeight?: number | string;
+  maxWidth?: number | string;
+  minHeight?: number;
+  minWidth?: number;
+  overflow?: string;
+  padding?: number;
+  paddingBottom?: number;
+  paddingHorizontal?: number;
+  paddingLeft?: number;
+  paddingRight?: number;
+  paddingTop?: number;
+  paddingVertical?: number;
+  position?: "relative" | "absolute";
+  reverse?: boolean;
+  row?: boolean;
+  width?: number | string;
+  display?: string;
+}
+
+const Box: React.FC<BoxProps> = ({
```

Comment:
duplicated while you can simply move this to the `/components` folder and use it in both places